DigitalLearningSolutions.Web/Views/LearningPortal/Current/_CurrentLearningResourceCard.cshtml

```diff
@@ -0,0 +1,46 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.LearningPortal.Current
+@model LearningResourceCardViewModel
+
+<div class="searchable-element nhsuk-panel @Model.DateStyle()" id="@Model.Id-lhr-card">
+  <details class="nhsuk-details nhsuk-expander nhsuk-u-margin-bottom-0">
+    <summary class="nhsuk-details__summary nhsuk-u-padding-top-0 nhsuk-u-padding-left-0">
+      <span class="nhsuk-u-visually-hidden">@Model.DueByDescription()</span>
+      <span class="nhsuk-details__summary-text searchable-element-title" id="@Model.Id-name-lhr" name="name" data-name-for-sorting="name" role="heading" aria-level="2">
+        @Model.Name
+      </span>
+    </summary>
+    <div class="nhsuk-details__text nhsuk-u-padding-left-0">
+      <div class="nhsuk-u-padding-left-3">
+        <div class="nhsuk-grid-row">
+          <partial name="Shared/_TagRow" model="@Model" />
+        </div>
+        <div class="nhsuk-grid-row">
+          <div class="nhsuk-grid-column-full">
+            @Html.Raw(Model.ResourceDescription)

```

Comment:
Haven't seen this used before, why this over @Model.ResourceDescription?

Reply:
The resource descriptions returned from the API calls sometimes seem to be written in HTML format, e.g. "< p> Test resource ... </ p>". Just displaying @Model.ResourceDescription would cause the <p> to be displayed as plain text. Html.Raw() allows the contents to be rendered as actual HTML so we can keep whatever formatting is stored with the description.